README.md

```diff
@@ -69,6 +69,7 @@ A Microverse Perosnal Portfolio Project developed with duo of HTML and CSS. In a
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 <a href="https://codeeq-microverse-portfolio.netlify.app/">Live Demo Link</a>
```

Comment:
- [OPTIONAL] Please remove this netlify link from the README because you already have provided the `gh-pages` link which is enough and as per the requirement. Kindly remove this link.